nav2_behavior_tree/include/nav2_behavior_tree/plugins/condition/is_stuck_condition.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2020 Samsung Research America
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_BEHAVIOR_TREE__PLUGINS__CONDITION__IS_STUCK_CONDITION_HPP_
+#define NAV2_BEHAVIOR_TREE__PLUGINS__CONDITION__IS_STUCK_CONDITION_HPP_
+
+// Copyright (c) 2018 Intel Corporation
```

Comment:
Combine the 2 licenses with intels' name before Samsung

Reply:
Ah, sorry missed this completely. The original `is_stuck_condition` had only intel's license, the samsung stuff got copied here by mistake. I'll remove the samsung license altogether.